tests/test_h2_client.c

```diff
@@ -22,6 +22,8 @@
     AWS_TEST_CASE(NAME, s_test_##NAME);                                                                                \
     static int s_test_##NAME(struct aws_allocator *allocator, void *ctx)
 
+#define MAX_FRAME_SIZE 16384
```

Comment:
delete this, you didn't use it
and once we're handling SETTINGS properly, the max frame size will be variable